### PR TITLE
Added option to attach an ELB to Opsworks layer

### PR DESCRIPTION
**Pull request by jorjao81**

### CODE REVIEW

File: builtin/providers/aws/opsworks_layers.go [hunk 1]
```diff
@@ -288,6 +293,29 @@ func (lt *opsworksLayerType) Read(d *schema.ResourceData, client *opsworks.OpsWo
...
 
+	/* get ELB */
+	ebsRequest := &opsworks.DescribeElasticLoadBalancersInput{
+		LayerIds: []*string{
+			aws.String(d.Id()),
+		},
+	}
+	loadBalancers, err := client.DescribeElasticLoadBalancers(ebsRequest)
+
+	fmt.Println(loadBalancers)
```
Comment (catsby) [inline builtin/providers/aws/opsworks_layers.go]: debug statement, I assume ? :smile:

File: builtin/providers/aws/opsworks_layers.go [hunk 2]
```diff
@@ -328,6 +356,15 @@ func (lt *opsworksLayerType) Create(d *schema.ResourceData, client *opsworks.Ops
 	d.SetId(layerId)
 	d.Set("id", layerId)
 
+	loadBalancer := aws.String(d.Get("elastic_load_balancer").(string))
+	if loadBalancer != nil && *loadBalancer != "" {
+		log.Printf("[DEBUG] Attaching load balancer: %s", *loadBalancer)
+		client.AttachElasticLoadBalancer(&opsworks.AttachElasticLoadBalancerInput{
```
Comment (catsby) [inline builtin/providers/aws/opsworks_layers.go]: We should inspect the error here and respond accordingly. If it failed, we should return the error at least.

**Comment by catsby** [comment]
Thanks @jorjao81 – this looks good, just 2 comments I'd like to see addressed and we're good here :smile:

**Comment by jorjao81** [comment]
Thanks for the comments @catsby . Let me know if there is anything else I should do to get this merged?

**Comment by catsby** [comment]
@jorjao81 nope, this should be good to merge, but I'm holding off for a minute to minimize something... soon :smile: 

Fixes #3778

**Comment by catsby** [comment]
Hey @jorjao81 – I pulled this locally to do a rebase, and pushed those up, thanks!

For reference, the new commits:

- https://github.com/hashicorp/terraform/commit/7293ab99d362b58aa6992cef7df2895c8c10a5dc
- https://github.com/hashicorp/terraform/commit/923f7dca37bfce57bb5b5da2046b040ead1c9b67

And I added this one:

- https://github.com/hashicorp/terraform/commit/320dc3ddf46fdd2531635cf99a6a4ddd78d5af72

It looks off because I had to re-`gofmt` after the rebase, but I also added the error check on line 421

**Pull request closed without merging**